module-1/cdk/lib/Module1Construct.ts

```diff
@@ -0,0 +1,74 @@
+import cloudfront = require("@aws-cdk/aws-cloudfront");
+import iam = require("@aws-cdk/aws-iam");
+import s3 = require("@aws-cdk/aws-s3");
+import s3deploy = require("@aws-cdk/aws-s3-deployment");
+import cdk = require("@aws-cdk/cdk");
+
+export class Module1Construct extends cdk.Construct {
```

Comment:
Since constructs are such a core primitive of the CDK, we treat them almost like a language primitive. We don't use a "Construct" postfix for constructs (it's like you won't add a "Class" postfix for all classes).

Reply:
Out of date - Now all stacks